crates/nu-command/tests/commands/alias.rs

```diff
@@ -60,7 +61,17 @@ fn alias_fails_with_invalid_name() {
     ));
     assert!(actual
         .err
-        .contains("alias name can't be a number or a filesize"));
+        .contains("alias name can't be a number, a filesize, or contain a hash"));
+
+    let actual = nu!(
+        cwd: "tests/fixtures/formats", pipeline(
+        r#"
+            alias "te#t" = echo "test"   
+        "#
+    ));
+    assert!(actual
+        .err
+        .contains("alias name can't be a number, a filesize, or contain a hash"));
```

Comment:
we could probably say something like "alias name can't start with a number or contain a hash"

Then prevent anything that starts with a number, in case we add support for more number+unit in the future.

Reply:
The issue I found with that was the command `7z`, which is still widely used. If we just disallow anything that starts with a number, custom completions can't be written for that, so I feel just updating when support is added for more number+unit is still better for the end-user.